[pytorch] Update troubleshooting_url

brad-mengchi opened the pull request:
Summary: Update new troubleshooting_url. Old one does not exist.

Test Plan: None

Differential Revision: D42205626



cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

soumith commented on torch/_dynamo/utils.py:
```diff
@@ -37,9 +37,7 @@
-troubleshooting_url = (
-    "https://github.com/pytorch/torchdynamo/blob/main/TROUBLESHOOTING.md"
-)
+troubleshooting_url = "https://github.com/pytorch/pytorch/blob/master/docs/source/dynamo/troubleshooting.rst"
```
thanks for fixing this. the correct link is https://pytorch.org/docs/master/dynamo/troubleshooting.html

brad-mengchi committed:
[pytorch] Update troubleshooting_url (#91298)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/91298

Update new troubleshooting_url. Old one does not exist.

Test Plan: None

Reviewed By: jianyuh

Differential Revision: D42205626

fbshipit-source-id: 8a3ae6a720b526fdf173ba2077eb85d7d61de41c
